common/build/eslint-config-fluid/minimal.js

```diff
@@ -168,7 +167,6 @@ module.exports = {
 		"@typescript-eslint/indent": "off",
 		"@typescript-eslint/member-ordering": "off",
 		"@typescript-eslint/no-explicit-any": "off",
-		"@typescript-eslint/no-parameter-properties": "off",
```

Comment:
I can't figure out what's up with this one. Is it now disabled by default? Deprecated? (If so, I couldn't find it in its documentation). I know we currently do this so wanted to make sure we're not causing new linter errors.

Reply:
According to the PR description, this rule was removed from the `typescript-eslint` plugin.